fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/block/entity/FabricBlockEntityTypeBuilder.java

```diff
@@ -31,23 +35,36 @@
  */
 public final class FabricBlockEntityTypeBuilder<T extends BlockEntity> {
 	private final Factory<? extends T> factory;
-	private final Block[] blocks;
+	private final List<Block> blocks;
 
-	private FabricBlockEntityTypeBuilder(Factory<? extends T> factory, Block[] blocks) {
+	private FabricBlockEntityTypeBuilder(Factory<? extends T> factory, List<Block> blocks) {
 		this.factory = factory;
 		this.blocks = blocks;
 	}
 
 	public static <T extends BlockEntity> FabricBlockEntityTypeBuilder<T> create(Factory<? extends T> factory, Block... blocks) {
-		return new FabricBlockEntityTypeBuilder<>(factory, blocks);
+		List<Block> blocksList = new ArrayList<>(blocks.length);
+		Collections.addAll(blocksList, blocks);
+
+		return new FabricBlockEntityTypeBuilder<>(factory, blocksList);
+	}
+
+	public FabricBlockEntityTypeBuilder<T> addBlock(Block block) {
```

Comment:
```suggestion
	/**
	 * Adds a supported block for the block entity type.
	 *
	 * @param block the supported block
	 * @return this builder
	 */
	public FabricBlockEntityTypeBuilder<T> addBlock(Block block) {
```
Hope the code style is correct, did this in the comment editor :smile:

Reply:
Fixed in d6051ccc02983ec66cfd541b054748091e3419d0